lib/ffmpeg/encoderbase.py

```diff
@@ -187,12 +188,15 @@ def encode(self):
 
     get_media().test_call_timeout = vars(self).get("call_timeout", 0)
 
-    iopts = self.gen_input_opts()
-    oopts = self.gen_output_opts()
     name  = self.gen_name().format(**vars(self))
     ext   = self.get_file_ext()
 
     self.encoded = get_media()._test_artifact("{}.{}".format(name, ext))
+    self.osencoded = filepath2os(self.encoded)
```

Comment:
Also need to do this again after line 207 (i.e. after `self.encoded = get_media()._test_artifact("{}_{}.{}".format(name, i, ext))`) for r2r cases

Reply:
Thanks, already modified